pkg/cmd/version/cmd.go

```diff
@@ -0,0 +1,25 @@
+package version
+
+import (
+	"fmt"
+	"strings"
+	"github.com/blang/semver"
+	"github.com/spf13/cobra"
+)
+
+var (
+	Raw = "v0.0.1"
+	Version = semver.MustParse(strings.TrimLeft(Raw, "v"))
+	String = fmt.Sprintf("ConsoleOperator %s", Raw)
+)
+
+func NewVersion() *cobra.Command {
```

Comment:
This is not useful - you need `git` info like `pkg/console/version/version.go`

Reply:
Agree, quick stub in, ill have to look into how that is done & how it maps to our releases since we want things formatted specifically.